### PR TITLE
mappings: fix folder structure for mappings

### LINKED ISSUE #13: mappings: fix folder structure for mappings
```
invenio-marc21/mappings/
invenio-marc21/mappings/marc21.json	
invenio-marc21/mappings/marc21_authority.json
invenio-marc21/mappings/marc21_holdings.json
```

-> 

```
invenio-marc21/mappings/
invenio-marc21/mappings/marc21bibliographic/marc21bibliographic-v1.0.0.json	
invenio-marc21/mappings/marc21authority/marc21authority-v1.0.0.json
invenio-marc21/mappings/marc21holdings/marc21holdings-v1.0.0.json
```

### PR DESCRIPTION
**Pull request by hachreak**
* Fixes folder structure for mappings. (closes #13)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
ping @lnielsen

**Comment by switowski** [comment]
Waiting for a sign from @lnielsen

File: tests/conftest.py [hunk 1]
```diff
@@ -35,8 +35,9 @@
 from flask_babelex import Babel
 from flask_cli import FlaskCLI
 from invenio_db import InvenioDB, db
+from invenio_jsonschemas import InvenioJSONSchemas
```
Comment (lnielsen) [inline tests/conftest.py]: Do you need to add it to test requirements in setup.py?

**Comment by lnielsen** [comment]
Please check comment, otherwise it LGTM

File: tests/conftest.py (hunk 1 above)
Comment (hachreak) [inline tests/conftest.py]: actually, it's already in `install_requires`.

File: tests/conftest.py [hunk 2]
```diff
@@ -66,14 +69,19 @@ def es_app(request):
     InvenioRecords(app)
     InvenioMARC21(app)
 
-    sleep(10)
+    search = InvenioSearch(app)
+
```
Comment (jirikuncar) [inline tests/conftest.py]: IMHO you can remove the empty lines around `search = ...`

File: tests/conftest.py [hunk 3]
```diff
@@ -66,14 +69,19 @@ def es_app(request):
...
+    InvenioJSONSchemas(app)
 
     with app.app_context():
         db.create_all()
+        [s for s in search.create()]
```
Comment (jirikuncar) [inline tests/conftest.py]: `list(search.cleate())`

**Comment by jirikuncar** [comment]
small cosmetics otherwise you can :shipit:

**Comment by hachreak** [comment]
done.. :shipit:  :)

**Comment by switowski** [comment]
:volcano: Merged, sorry for the delay :blush:

**Pull request merged**